users/management/commands/deadname_removal.py

```diff
@@ -0,0 +1,213 @@
+from django.core.management.base import BaseCommand
+
+from newsletters.models import Newsletter
+from pages.models import Page
+from minutes.models import Meeting
+
+from enum import Enum
+from titlecase import titlecase
+import re
+
+class CapitalisationType(Enum):
+    LOWERCASE = 1
+    UPPERCASE = 2
+    TITLECASE = 3
+
+class MatchLabel():
+    capitalisation : CapitalisationType = CapitalisationType.LOWERCASE
+    apostrophe = False
+
+    def __init__(self, cap, apos):
+        self.capitalisation = cap
+        self.apostrophe = apos
+
+def to_capitalisation(label, target):
+    captype = label.capitalisation
```

Comment:
gonna ask for user to provide the input in TitleCase, so that we have the more complex capitalisation to hand (although as a bonus might be smart to check if it's all lowercase, and if so apply `titlecase`)